cohort_4/playbook/ACTIVITIES.md

```diff
@@ -551,6 +551,16 @@ then).
 
 # Workshops
 
+## General Guidelines
+
+### Breaks
+
+If hackers don't take a break, they will likely get very antsy. You can either 
+
+- remind them to take breaks every hour / hour and a half
```

Comment:
@zachlatta we're not, that is one alternative, the other is just to say it once because from observation, people don't take breaks.